http/resp/responder.go

```diff
@@ -409,6 +418,49 @@ func (doer *Responder) do(w http.ResponseWriter, r *http.Request, opts ...Fn) (*
 	return resp, nil
 }
 
+// handleHtmlError specially renders the error template set on the Responder
+// and reports errors.
+func (doer *Responder) handleHtmlError(w http.ResponseWriter, r *http.Request, err error) error {
+	w.WriteHeader(http.StatusInternalServerError)
+
+	if doer.templates.err == "" {
+		err = fmt.Errorf(
+			"%w: no error template provided, encountered while handling: %s",
+			ErrBadConfig,
+			err,
+		)
+		return err
+	}
+	b := doer.pool.Get().(*bytes.Buffer)
```

Comment:
Not super relevant to this PR, but I feel like the pool adds complexity with rendering without really solving anything for our use cases. It would be one thing if we went to the pool because we were hitting some thresholds.